PySimpleSQL/pysimplesql
# Hello World, mkdocs / mkdocstrings

To test, you'd need to:

pip install:
mkdocs
mkdocstrings
mkdocstrings-python
mkdocs-material

then cd into pysimplesql and run:
`mkdocs serve`

I’ll start breaking things out into pages, and only include functions useful for development.

When we are ready, we can either host on GitHub pages, or read the docs.
See https://www.mkdocs.org/user-guide/deploying-your-docs/
